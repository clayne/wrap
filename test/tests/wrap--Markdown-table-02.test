wrap | /dev/null | -u | md-table-02.md | 0
