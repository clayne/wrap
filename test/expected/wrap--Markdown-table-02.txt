This is a line of text.

Column 1 | Column 2
---------|---------
Entry 1  | Entry 2

This is a second line of text.  This is a third line of text.  This is a fourth
line of text.
